src/main/java/dk/dbc/opensearch/OpensearchQuery.java

```diff
@@ -156,6 +171,9 @@ public String build() throws java.io.UnsupportedEncodingException {
         if(this.marc001b != null && !this.marc001b.isBlank()) {
```

Comment:
Ved godt at denne ikke er en del af branchens rettelser, men den kan gøres bedre. Brugen af marc001b er kun egnet som en begrænser - dvs. ens generelle søgning begrænses til en bestemt base. Derfor vil OR aldrig være relevant da den vil trække en hel base med i søgeresultatet. En AND vil derfor altid være den der ønskes, men koblet sammen med at det øvrige søgeudtryk er indkapslet i paranteser. 
F.eks : 
(fo="karl smart" OR isbn=1234567890123) AND marc.001b=870970